applications_tests/lethe-particles/insert_plane_3d.output

```diff
@@ -40,4 +40,4 @@ id, type, dp, x, y, z
 28 0 0.01000 0.2079 -0.0069 0.3979
 29 0 0.01000 -0.0357 -0.0020 -0.3063
 30 0 0.01000 0.2206 0.0303 0.1907
-31 0 0.01000 -0.0391 0.0187 0.2174
+31 0 0.01000 -0.0391 0.0187 0.2174
```

Comment:
No change?

Reply:
I think is was an empty line after the last line or something.